print/static/print-preview.css

```diff
@@ -66,6 +66,7 @@
 
     /* uncomment this part for recto/verso book : ------------------------------------ */
 
+    /* stylelint-disable no-duplicate-selectors, no-descending-specificity */
```

Comment:
@usu had to disable some rules here to make stylelint happy, I didn't want to tinker with your CSS 😉